exercise_015_clustered_sudoku_solver/src/test/resources/README.md

```diff
@@ -1,5 +1,7 @@
-clustered_sudoku_solver
+base_clustered_sudoku_solver
 
 # Clustered Sudoku Solver
 
-In this step, we add an actor based Sudoku solver. Each node will automatically start a solver and another actor will send the same sudoku problem in a continues loop.
\ No newline at end of file
+In this step, we add an actor based Sudoku solver. Each node 
```

Comment:
Why did you do explicit CRLF's here. Just curios

Reply:
The content of the README.md files is displayed (and formatted) by the man [e] command. I try to pre-format it to a ±80 char wide text block for better displaying. On of the reasons for doing this is that the pseudo markDown formatter in the tools (course-management-tools) doesn't have built-in support for formatting list (i.e correct indentation).